microsoft-edge/extensions-chromium/whats-new/released-features.md

```diff
@@ -17,6 +17,15 @@ This article describes features launched on [Partner Center](https://partner.mic
 For more information about upcoming updates and feature releases, see [Microsoft Edge Add-ons roadmap](roadmap.md).
 
 
+
+<!-- ====================================================================== -->
+### Developers can respond to user reviews about their Microsoft Edge add-on
```

Comment:
@vchapel  - pls consider a change maybe in other PR, do full-text search trying case-sensitive & trying Whole Word: finding: 

"Add-on" is almost never in the singular, use extension instead? 

"add-on" is almost never lowercase; change to always uppercase?

Reply:
Right, fixing